src/fstab-generator/fstab-generator.c

```diff
@@ -858,6 +871,8 @@ static MountPointFlags fstab_options_to_flags(const char *options, bool is_swap)
                 flags |= MOUNT_GROWFS;
         if (fstab_test_option(options, "x-systemd.pcrfs\0"))
                 flags |= MOUNT_PCRFS;
+        if (fstab_test_option(options, "quota\0" "usrquota\0"  "grpquota\0"  "prjquota\0"))
```

Comment:
the old logic tested for  "usrquota\0" "grpquota\0" "quota\0" "usrjquota\0" "grpjquota\0"

which is different from the list you have here? whic hone is right?